src/app/components/SwiperGallery/SwiperGallery.tsx

```diff
@@ -0,0 +1,29 @@
+import Image from "next/image";
+import { swiperGalleryData } from "@/app/utils/swiperGalleryData";
+import Container from "../Share/Container";
+import { Swiper } from "../Share/Swiper/Swiper";
+import { SwiperGalleryTitle } from "./SwiperGalleryTitle";
+
+export const SwiperGallery = () => {
+  return (
+    <section id="gallery" className="pb-12 pt-[70px] desktop:pt-[88px]">
+      <Container>
```

Comment:
You need to add "O nas / Galeria" on the top. Like on Figma template :) Marlena forgot this, and on mobile the section name is. Let's stick to a consistent design :)